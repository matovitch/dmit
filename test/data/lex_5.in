2 * y
_Test_42-12 10.79e-17
A.B