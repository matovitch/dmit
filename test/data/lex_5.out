{"tokens" :[{"token":"START_OF_INPUT"},{"token":"INTEGER"},{"token":"WHITESPACE"},{"token":"STAR"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"MINUS"},{"token":"INTEGER"},{"token":"WHITESPACE"},{"token":"DECIMAL"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"DOT"},{"token":"IDENTIFIER"},{"token":"END_OF_INPUT"}],"offsets":[31,30,29,28,27,26,25,17,16,14,13,4,3,2,1,0,0]}